javagen/src/main/java/com/azure/autorest/mapper/ModelMapper.java

```diff
@@ -29,13 +30,13 @@ public static ModelMapper getInstance() {
     @Override
     public ClientModel map(ObjectSchema compositeType) {
         JavaSettings settings = JavaSettings.getInstance();
-        ClientModel result = serviceModels.getModel(compositeType.getLanguage().getJava().getName());
-        if (result == null) {
-            String modelSubPackage = !settings.isFluent() ? settings.getModelsSubpackage() : (false /*compositeType.IsInnerModel*/ ? settings.getImplementationSubpackage() : "");
-            if (settings.isCustomType(compositeType.getLanguage().getJava().getName())) {
-                modelSubPackage = settings.getCustomTypesSubpackage();
-            }
-            String modelPackage = settings.getPackage(modelSubPackage);
+        ObjectMapper objectMapper = Mappers.getObjectMapper();
+
+        ClassType modelType = objectMapper.map(compositeType);
+        String modelName = modelType.getName();
```

Comment:
I am trying to make result from ObjectMapper as the ground truth for ObjectSchema, at least on class name + package name.